app/test_split_integer.py

```diff
@@ -2,20 +2,21 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    parts = split_integer(32, 6)
```

Comment:
The variable `parts` is unnecessary here. According to the checklist, it's better to directly use the function call in the assert statement: `assert sum(split_integer(32, 6)) == 32`.